core/app/models/spree/order.rb

```diff
@@ -168,6 +165,10 @@ def self.register_line_item_comparison_hook(hook)
       line_item_comparison_hooks.add(hook)
     end
 
+    def to_param
+      number.to_s
```

Comment:
@damianlegawiec but we should use [`to_param`](http://guides.rubyonrails.org/active_support_core_extensions.html#to-param) and leave it in particular models
https://github.com/norman/friendly_id/blob/aff0564584e84ffa505e6e278b65ca3c4ee5d698/lib/friendly_id/base.rb#L259-L270